hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -4133,9 +4156,61 @@ func reconcileVirtLauncherNetworkPolicy(policy *networkingv1.NetworkPolicy, hclu
 			},
 		},
 	}
-	policy.Spec.PodSelector = metav1.LabelSelector{
-		MatchLabels: map[string]string{
-			"kubevirt.io": "virt-launcher",
+	policy.Spec.Egress = []networkingv1.NetworkPolicyEgressRule{
+		{
+			To: []networkingv1.NetworkPolicyPeer{
+				{
+					// Allow access towards outside the cluster for the virtual machines (guest nodes),
+					// But deny access to other cluster's pods and services with the exceptions below
+					IPBlock: &networkingv1.IPBlock{
+						CIDR:   "0.0.0.0/0",
```

Comment:
We have to support dual stack too, so we need another with "::/0"

Reply:
if the cluster has configured with IPv6, the `managementNetwork.spec.clusterNetwork` array should contain the IPv6 ranges in addition to the IPv4 ones, right?
can we use the same `blockedNetworks` list that contains both IPv4 and IPv6 for IPBlock of 0.0.0.0/0 and for IPBlock of ::/0 ?

Reply:
After some talks with networking team looks like we need to also split `blockedNetworks` into `blockedIPv4Networks` and `blockedIPv6Networks` 

```golang

```